### ISSUE TITLE
Convert user-submitted URLs before save

### ISSUE THREAD

**Issue by joshsmith** [code-corps/deprecated-code-corps-rails-api#383]
Should do something like the following:

``` ruby
before_save :add_url_protocol

private

def add_url_protocol
  unless self.website[/^http?:\/\//] || self.website[/^https?:\/\//]
    self.website = "http://#{self.website}"
  end
end
```

**Comment by joshsmith** [comment]
We'll need to add a migration to fix this with existing websites, too.

**Comment by joshsmith** [comment]
Closed by #395.
